tests/bluetooth/tester/src/model_handler.c

```diff
@@ -0,0 +1,608 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```

Comment:
Should be `SPDX-License-Identifier: LicenseRef-Nordic-5-Clause`, in accordance with #3842